### DISCUSSION TITLE
[SUPPORT] Spain/Nordic two-factor autentication

### DISCUSSION THREAD

**Discussion by tomjimlondon** [arvvoid/plugin.video.hbogoeu#208] [Q&A]
Hello team,

Good work with the add-on! It seems that recently HBO Spain request a double factor to add a new device. It should appear a code on the device's screen that must be added on their website in order to be authorised. Can you implement it to keep working? 

Thanks in advance.

**Comment by arvvoid** [comment]
Hello, this add-on works in similar way as navigating to the hbo website with a browser it just present the data from within Kodi. So if access to the website work as before no changes are necessary probably. Just insert the credentials the same way that you login on the website. Have you tried to login with the add-on?

If the  login stopped working in the add-on full debug logs are necessary. And probably someone from the region with access will have to implement the changes and submit  a pull request. Or send detailed browser logs.

**Comment by tomjimlondon** [comment]
Here you are:
https://paste.kodi.tv/oxegidizij

**Comment by arvvoid** [comment]
@tomjimlondon that is not a debug log, please enable debug loggin in Kodi first. Link to instructions can be found in the readme. Then perform some actions in the add-on and then send the log.

`2021-01-24 13:34:15.978 T:16963  NOTICE: Log level changed to "LOG_LEVEL_NORMAL"`

**Comment by arvvoid** [comment]
Closing, since there are no new updates on the matter and is unclear if currently there are any problems with login for the sp/no region. By now there should be plenty of reports if the login is broken.

If the login is broken for sp/no a bug report can be submitted with a full debug log.
